### DISCUSSION TITLE
Can we use this project with awesome gui projects like open-webui ? if yes then how please guide on this

### DISCUSSION THREAD

**Discussion by Greatz08** [mem0ai/mem0#1513] [General]
Recently been using ollama and claud and others supported by Litellm with open-webui so was curious if it is possible or not

**Comment by bwuuu** [comment]
Also curious here. OWUI recently added streamable HTTP support for MCP in addition to OpenAPI. If OpenMemory would somehow support streamableHTTP then it would be perfect.
